src/app/navigation/header/header.component.html

```diff
@@ -1,8 +1,33 @@
 <div class="header">
-  <button mat-button routerLink="home">Home</button>
-  <button mat-button routerLink="admin">Admin</button>
-  <button mat-button routerLink="login">Sing-in</button>
-  <button mat-icon-button (click)="openSidebar()">
-    <mat-icon>person</mat-icon>
-  </button>
+  <mat-toolbar>
+    <app-logo routerLink="home" (click)="adminPageVisible = false"></app-logo>
+    <span class="header__spacer"></span>
+    <nav class="header__nav">
+      <button
+        class="button button_l"
+        mat-button
+        *ngIf="!adminPageVisible; else homePageButton"
+        routerLink="admin"
+        (click)="toggleAdminPage()"
+      >
+        Manage
+      </button>
```

Comment:
it's a new page, it should be a link ```<a>```

Reply:
I`m not sure of this. Here I use material-buttons, they are already well styled.

Reply:
```<a mat-raised-button href="#">Link</a>```

https://material.angular.io/components/button/overview